### ISSUE TITLE
Allow for only setting Minimum CPU / GPU Clock(s) with no Maximum

### ISSUE THREAD

**Issue by syberphunk** [NGnius/PowerTools#106]
### Please confirm

- [X] I have searched existing issues
- [X] This issue is not a duplicate of an existing one
- [X] I will fill this out to the best of my ability

### Describe what you'd like to be able to do

Steam Deck users are overclocking the GPU and CPU using tools such as https://github.com/DavidS95/Smokeless_UMAF

PowerTools allows us to configure an override to accommodate these higher values for GPU and CPU clock, but a current bug with amdgpu prevents us from setting higher than stock clock speeds.

If we set no maximum value, the Steam Deck with these overclocks in place will automatically scale upwards.

Unfortunately we cannot currently set only a 'minimum' without also setting a 'maximum' and take advantage of these overclock values.

Some games, emulators, etc. need minimum clock values to be set because the automatic scaling of its frequency may not be appropriate for the game or software that's being run, even if it won't scale higher than this minimum value.

Therefore it would be good to be able to set a minimum value, without enforcing a maximum value for the clock speeds.

It would be good to be able to set this option of 'no maximum values for clocks' in the user interface, else editing pt_oc.json would be an alternative workaround to permit it.

### Describe alternatives you've considered

There's no real other alternative aside from not setting clock minimum/maximum at all per game, and this removes the ability to hold the APU accountable to run at least at a minimum speed.

**Comment by NGnius** [comment]
There's always equivalent commands

**Comment by syberphunk** [comment]
Having the controls via GUI is convenient and the whole purpose of having decky plugins.

**Comment by NGnius** [comment]
Agreed, my suggestion was strictly in the context of being a workaround. There is also a Decky plugin for running commands.

**Comment by syberphunk** [comment]
Noting this as related: https://github.com/badly-drawn-wizards/vangogh_oc_fix as observed by JDGBOLT. Not suggesting this is within scope of PowerTools.

**Comment by badly-drawn-wizards** [comment]
Just chipping in to say that I have filed an issue. Although I have little hope that it will be implemented and if it does it will be a long while until it trickles down to SteamOS: 
https://gitlab.freedesktop.org/drm/amd/-/issues/2638

I hope my fix becomes irrelevant as fast as possible. :pray:
